environments/india/app-processes.yml

```diff
@@ -36,7 +36,7 @@ celery_processes:
     background_queue,dashboard_comparison_queue:
       concurrency: 2
 pillows:
-  pillow2:
+  pillow2,pillow3:
```

Comment:
This might now work the way you think because pillow machines are much less flexible in their routing than celery machines. It might be simpler if you just replace pillow2 with pillow3

Reply:
I've removed these changes and I'll replace the definitions in a later commit.